fr/contributing/documentation.rst

```diff
@@ -369,15 +370,16 @@ Chaque directive remplit l'index, et l'index des espaces de nom.
 
    Décrit une propriété/attribut sur une classe.
 
-Prevent Sphinx to Output Warnings
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Eviter l'Affichage d'Avertissements de Sphinx
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Sphinx will output warnings if a function is referenced in multiple files. It's
-a great way to ensure that you did not add a function two times, but
-sometimes, you actually want to write a function in two or more files, eg.
-`debug object` is referenced in `/development/debugging` and in
-`/core-libraries/global-constants-and-functions`. In this case, you can add
-``:noindex:`` under the function debug, and the warnings will stop to alert you::
+Sphinx va afficher des avertissements si une fonction est référencée dans
+plusieurs fichiers. C'est un bon moyen de s'assurer que vous n'avez pas ajouté
+une fonction deux fois, mais parfois vous voulez en fait écrire une fonction
+dans deux ou plusieurs fichiers, par exemple `debug object` est référencé dans
+`/development/debugging` et dans
+`/core-libraries/global-constants-and-functions`. Dans ce cas, vous pouvez
+ajouter ``:noindex:`` sous la fonction debug pour supprimer les avertissements::
```

Comment:
Yes, your last sentence is correct !
You have to keep only one of the methods **WITHOUT** `:noindex:`. I'll try to make it more accurate